.github/workflows/test.yml

```diff
@@ -188,3 +188,17 @@ jobs:
         fail_ci_if_error: true
         files: ./coverage.xml
         verbose: true
+
+  check:  # This job does nothing and is only used for the branch protection
```

Comment:
Let's give it a better name than check

Reply:
Well, PyCA uses `all-green`, some use `all-successful`. I originally was going to recommend `gate` but a colleague convinced me that `check` is less confusing (because it doesn't fully map to https://gating.dev and openstack's understanding of gating).
This is why I left `check` in the action readme example so most people would just copy-paste that. For consistency, I'd have the same job id across the board but maybe specify a differing `name:` value?

Reply:
Different name is fine, it's just thst check is absolutely useless in terms of deriving meaning given gh naming schemes

Required-checks-agregate would be far more telling for example

Also there is possibly a need to limit who can change the dependencies

Reply:
Okay, I'll let the maintainers bikeshed on the name, then.
As for updating branch protection in the repo settings, it's usually available for the repo admins, I think.

Reply:
Ill give it a spin this evening